### DISCUSSION TITLE
Creating an database-based ontology using MAUI app

### DISCUSSION THREAD

**Discussion by ychaix** [CommunityToolkit/Maui#1396] [General]
I am 80+ and starting to relearn to program, C# with WPF. Was an SOA specialist for some 10 years (analyst, architect, governance and security with Thomas Erl Arcitura). I started conceptualizing an inventory system back in 1980´s, under MS-DOS, Clipper and dBase. Eventually evolved the taxonomy into a **_database-based medium weight ontology_** to publish and make available publicly everything industry related. Presently the taxonomy has almost 30,000 categories (taxa), close to 10,000 taxa instances. Somebody at IOF (Industry Ontology Foundry) argues that ontologies do not hold instances. I strongly disagree and I have since stopped aspiring at being invited to join IOF.
I tried to interest some not so young IT friends, but I am not getting anywhere, particularly since I want to develop the application as a WPF Entity framework, MAUI style and they are not really into it.
I have created most of the use cases for the application (a mixture of spanish and english), the database is on SQLServer.  The whole documentation is presently 2.2GB. 
I reckon I have about 10 yrs of shelflife left and feel a hurry to try and create this and not let it go to waste. Potentially, the users for this application would be any SME around the world and most engineering cabinets. 
I am pretty good at the analysis and REST and Web services part (and just learnt yesterday how to create a recursive query on my catologue-tree) but not so much at programming. It is kind of a low process at 80. I have started studying WPF Simplified by Mahesh Chand and find it quite good.
Anybody interested, particularly for the programming part? I did not find any similar discussion.

**Comment by ychaix** [comment]
Sorry, I just realized that the discussions object of this site are related to feature proposals for MAUI. I do not know how to remove my post, so please ignore it.

**Comment by CliffAgius** [comment]
Closed and answered
